tests/integration/docker/test.common

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+DOCKER_EXE="docker"
+DOCKER_SERVICE="docker-cor"
+
+#Cleaning test environment
+function cleanDockerPs(){
+	"$DOCKER_EXE" ps -q | xargs -r "$DOCKER_EXE" kill
+	"$DOCKER_EXE" ps -aq | xargs -r "$DOCKER_EXE" rm -f
+}
+
+#Restarting test environment
+function startDockerService(){
+	systemctl status "$DOCKER_SERVICE" | grep 'running'
+	if [ "$?" -eq 0 ]; then
+		systemctl restart "$DOCKER_SERVICE"
+	fi
+}
+
+#Checking that default runtime is cor
+function runtimeDocker(){
+    "$DOCKER_EXE" info | grep "Default Runtime: cor"
```

Comment:
same as docker service, this runtime id that is registered in dockerd is the same in all obs packages ?